src/extension.ts

```diff
@@ -231,6 +236,7 @@ async function activate(context: vscode.ExtensionContext) {
     }));
     context.subscriptions.push(vscode.window.onDidChangeTextEditorSelection(event => {
         replHistory.setReplHistoryCommandsActiveContext(event.textEditor);
+        checkForContextChange(event.textEditor);
```

Comment:
Since this isn't just a pure check and does some side-effect, and since it's specifically about the cursor, I think a name like `setCursorContext` might be better. It can still do the check before setting, etc.